test/unit/uk/gov/hmrc/tariffclassificationfrontend/service/CasesServiceSpec.scala

```diff
@@ -34,21 +36,22 @@ import scala.concurrent.Future
 
 class CasesServiceSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach {
 
-  private implicit val hc = HeaderCarrier()
+  private implicit val hc: HeaderCarrier = HeaderCarrier()
   private val manyCases = mock[Seq[Case]]
   private val oneCase = mock[Option[Case]]
   private val queue = mock[Queue]
   private val connector = mock[BindingTariffClassificationConnector]
+  private val audit = mock[AuditService]
```

Comment:
you should assert that the audit events have been created, using Mockito (as I did in the controller)

Reply:
actually I noticed you created two additional files for this service tests